blotztask-ui/src/app/task-list/components/task-table.tsx

```diff
@@ -1,46 +1,54 @@
 import {
-  Table,
-  TableBody,
-  TableCell,
-  TableHead,
-  TableHeader,
-  TableRow,
-} from '@/components/ui/table';
+  Card,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card';
+// import { TaskItemDTO } from '@/model/task-Item-dto';
 
-import { TaskItemDTO } from '@/model/task-Item-dto';
+// interface TaskTableProps {
+//   tasks: TaskItemDTO[]; // tasks prop is an array of TaskItemDTO objects
+// }
 
-interface TaskTableProps {
-  tasks: TaskItemDTO[]; // tasks prop is an array of TaskItemDTO objects
-}
-
-export function TaskTable({ tasks }: TaskTableProps) {
+export function TaskTable({ tasks }) {
```

Comment:
```suggestion
export function TaskCard({ tasks }) {
```

if this is now custom card component , you can change the name

Reply:
has changed the name